doc/source/math/basic.rst

```diff
@@ -63,11 +63,22 @@ constants about the universe that you may find handy in your math operations.
 
 .. global:: Constant:PI
 
-    Ratio of circumference of a circle to its diameter
+    Ratio of circumference of a circle to its diameter, 3.14159265...
+    
+    ::
+
+        SET diameter to 10.
+        PRINT "circumference is:".
+        PRINT constant:pi * diameter.
 
 .. global:: Constant:C
 
     Speed of light in a vacuum, in meters per second.
+    
+    ::
+
+        PRINT "Light goes this fast:".
+        PRINT constant:c.
```

Comment:
Speaking of an example not doing anything...

I agree, this example is nothing more than showing how to access a suffix.  If you're set on providing an example, maybe you could perform some math with it.

```
local speed is ship:velocity:orbit:mag.
local speedOfLight is constant:c.
local ratio is speed / speedOfLight.
print "Current speed is " + ratio + " times the speed of light.".
```
While still relatively meaningless, at least it gives the reader an example that does something.